ext/RMagick/rmimage.c

```diff
@@ -4158,12 +4158,14 @@ Image_contrast_stretch_channel(int argc, VALUE *argv, VALUE self)
 VALUE
 Image_morphology(VALUE self, VALUE method_v, VALUE iterations, VALUE kernel_v)
 {
-  static VALUE default_channels_const = 0;
+    static VALUE default_channels_const = 0;
 
-  if(!default_channels_const)
-    default_channels_const = rb_const_get(Module_Magick, rb_intern("DefaultChannels"));
+    if(!default_channels_const)
+    {
```

Comment:
I saw some places in the new code that was added where we didn't use braces for `if` blocks. Do we want to add braces everywhere? That would be my vote.

Reply:
I don't always add those braces but @Watson1978 told me in another PR to add those. I thought the convention of this project was to always use braces and that is why I added them here.

Reply:
I'm afraid someone will add wrong code like https://nakedsecurity.sophos.com/2014/02/24/anatomy-of-a-goto-fail-apples-ssl-bug-explained-plus-an-unofficial-patch/ :p